### PR TITLE
fix postgres tests

### PR DESCRIPTION
**Pull request by igorbernstein**

### CODE REVIEW

**Comment by igorbernstein** [comment]
not sure what you mean....the visualizer env is there:
https://github.com/igorbernstein/luigi/blob/fix-postgres-tests/.travis.yml#L31

**Comment by igorbernstein** [comment]
also: https://travis-ci.org/spotify/luigi/jobs/92587989

**Comment by igorbernstein** [comment]
hmm not sure why the visualizer env is failing on class parameter tests now...investigating

**Comment by igorbernstein** [comment]
oops, I see what you mean...I screwed up with the original rebase :(

**Comment by econchick** [comment]
@igorbernstein no worries, it's Sunday :)

File: test/postgres_with_server_test.py [hunk 1]
```diff
@@ -57,7 +57,6 @@
 
 # to avoid copying:
 
-@attr('postgres')
 class CopyToTestDB(postgres.CopyToTable):
```
Comment (econchick) [inline test/postgres_with_server_test.py]: Is there a reason to remove this attr?

**Comment by igorbernstein** [comment]
yep, it was misplaced...its used by nose to select which tests to run & needs to go on the test itself not on the luigi task

**Comment by Tarrasch** [comment]
I think I put the `attr()` annotation there and I can confirm that I had no idea what I was doing. :)

LGTM.

As a follow up PR, you might want to look at [this](https://github.com/spotify/luigi/blob/dd40a10296b1a4c4e508f5975acaf8163899fd16/.travis.yml#L45-L47). :)

**Comment by DeaconDesperado** [comment]
:+1: Nice work!

**Pull request merged**